src/views/LotteryV2/components/WinningNumbers.tsx

```diff
@@ -1,63 +1,20 @@
 import React from 'react'
-import styled from 'styled-components'
-import { Flex, Text, Box } from '@pancakeswap/uikit'
+import { Flex } from '@pancakeswap/uikit'
 import { parseRetreivedNumber } from '../helpers'
-import { PinkBall, LilacBall, TealBall, AquaBall, GreenBall, YellowBall } from '../svgs'
-
-const BallTextWrapper = styled.div`
```

Comment:
Balls.tsx now contains ready-to-use balls.

Reply:
Absolutely dreamy